packages/ui/src/machines/authenticator/actions.ts

```diff
@@ -139,10 +140,13 @@ export const handleInput = assign({
 });
 
 export const handleSubmit = assign({
-  formValues: (context, event: AuthEvent) => ({
-    ...context['formValues'],
-    ...event.data,
-  }),
+  formValues: (context, event: AuthEvent) => {
+    const formValues = {
+      ...context['formValues'],
+      ...event.data,
+    };
+    return trimValues(formValues, 'password');
```

Comment:
Nit: add a comment here or assign `password` to a constant (and add a comment 😆 )

Reply:
I think a comment suffices, will do